Source package : OpenMolcas
File name       : OpenMolcas-24.06.tar.bz2
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/OpenMolcas/OpenMolcas-24.06.tar.bz2/sha512/c200d450f6302b69cdbd220f37c52e57d53dbead6326378a8432179ee73ab54d8fae9b261bdffcb5d3e11401d49d65c7deff97fbaa2d8e822a89c53a58216449/OpenMolcas-24.06.tar.bz2
Source URL      : https://gitlab.com/Molcas/OpenMolcas/-/archive/v24.06/OpenMolcas-24.06.tar.bz2

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-OpenMolcas-24.06.tar.bz2: bzip2 compressed data, block size = 900k
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-OpenMolcas-24.06.tar.bz2: bzip2 compressed data, block size = 900k


diff -r source-OpenMolcas-24.06.tar.bz2-extracted/OpenMolcas-v24.06-ff0b5d1cb86ae8766625b9018d18260f07e46f52/.tags lookaside-OpenMolcas-24.06.tar.bz2-extracted/OpenMolcas-v24.06-ff0b5d1cb86ae8766625b9018d18260f07e46f52/.tags
2c2
<  (tag: v24.06, refs/keep-around/ff0b5d1cb86ae8766625b9018d18260f07e46f52)
---
>  (HEAD -> master, tag: v24.06, refs/merge-requests/688/head, refs/keep-around/ff0b5d1cb86ae8766625b9018d18260f07e46f52)
